### PR TITLE
Handle failed archive generation

### PR DESCRIPTION
**Pull request by timrogers**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -97,6 +97,9 @@ async function checkSingleMigrationStatus(migrationId: { org: string, migration_
             fileStream.end();
             console.log(`Migration ${migrationId.migration_id} archive downloaded to ${filePath}.`)
             return { migrationId: migrationId.migration_id, migrationStatus: migrationStatus };
+        } else if (migrationStatus === "failed") {
+           console.log(`ERROR: Archive generation failed for migration ${migrationId.migration_id}.`);
+           return { migrationId: migrationId.migration_id, migrationStatus };
```
Comment (pmartindev) [inline src/main.ts]: `migrationStatus` needs the key for the json value
`migrationStatus: migrationStatus`

Comment (timrogers) [reply src/main.ts]: Modern JavaScript should be okay with this - you don't need to specify the value if the key and value are the same 😊

Comment (timrogers) [reply src/main.ts]: This shows an example if you click "Run": https://jsfiddle.net/brL2pxyt/

Comment (pmartindev) [reply src/main.ts]: TIL :D! Tested and confirmed!

**Pull request merged**